legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/DataSpaceAnalyticsHelper.java

```diff
@@ -76,12 +80,22 @@
 import org.finos.legend.pure.m3.coreinstance.meta.pure.metamodel.function.LambdaFunction;
 
 import java.util.ArrayList;
+import java.util.List;
 import java.util.ServiceLoader;
 import java.util.stream.Collectors;
 
+import static org.finos.legend.engine.shared.core.ObjectMapperFactory.withStandardConfigurations;
+
+
 public class DataSpaceAnalyticsHelper
 {
-    private static final ObjectMapper objectMapper = ObjectMapperFactory.getNewStandardObjectMapperWithPureProtocolExtensionSupports();
+    private static final ObjectMapper objectMapper = EntitlementModelObjectMapperFactory.withEntitlementModelExtensions(withStandardConfigurations(PureProtocolObjectMapperFactory.withPureProtocolExtensions(new ObjectMapper())));
+    private static List<EntitlementServiceExtension> entitlementServiceExtensions = EntitlementServiceExtensionLoader.extensions();
+
+    public static ObjectMapper getObjectMapper()
```

Comment:
TODO: instead of making all 3 places use this object mapper, create an util `factory` function instead